python_modules/dagster-airflow/dagster_airflow/dagster_plugin.py

```diff
@@ -125,6 +139,83 @@
 # }
 
 
+LINE_LENGTH = 100
+
+# We include this directly to avoid taking the dependency on dagster
+class IndentingBlockPrinter(object):
```

Comment:
why do we need to avoid a dependency on dagster?

Reply:
can you write out a more explicit comment explaining where this copy was copied from and the rationale for doing so